src/igvfd/tests/test_schema_in_vitro_system.py

```diff
@@ -21,3 +21,40 @@ def test_time_post_factors_dependency(in_vitro_differentiated_cell, testapp):
         in_vitro_differentiated_cell['@id'],
         {'time_post_factors_introduction': 3, 'time_post_factors_introduction_units': 'day'}, expect_errors=True)
     assert res.status_code == 200
+
+
+def test_sorted_fraction(testapp, lab, award, source, human_donor, sample_term_K562):
+    item_1 = {
+        'classification': 'differentiated cell',
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'source': source['@id'],
+        'taxa': 'Homo sapiens',
+        'donors': [human_donor['@id']],
+        'biosample_term': sample_term_K562['@id']
+    }
+    source_cell = testapp.post_json('/in_vitro_system', item_1, status=201).json['@graph'][0]
+    item_2 = {
+        'classification': 'differentiated cell',
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'source': source['@id'],
+        'taxa': 'Homo sapiens',
+        'donors': [human_donor['@id']],
+        'biosample_term': sample_term_K562['@id'],
+        'sorted_fraction': source_cell['@id']
+    }
+    res = testapp.post_json('/in_vitro_system', item_2, status=201)
+    assert res.status_code == 201
+    item_3 = {
+        'classification': 'differentiated cell',
+        'award': award['@id'],
+        'lab': lab['@id'],
+        'source': source['@id'],
+        'taxa': 'Homo sapiens',
+        'donors': [human_donor['@id']],
+        'biosample_term': sample_term_K562['@id'],
+        'sorted_fraction': 'I am just pretending to be a sorted fraction.'
+    }
```

Comment:
while it is correct, I am not sure necessary test, the "sorted_fraction" is a link, so trying to post it with a string will fail..

Reply:
I want to make sure that the link is working correctly.  I am testing both success and failure of the link for the new schema property.  The test is changed over to bring in data from fixtures.